portfolio/src/main/java/com/google/sps/input/Comments.java

```diff
@@ -14,15 +14,14 @@
 
 package com.google.sps.input;
 
-/** Will be used in the next PR (just setting it up!) */
 public final class Comments {
   private final long id;
-  private final String title;
+  private final String entry;
```

Comment:
maybe call it message or comment? You can keep the current name if you want to.

Reply:
Done